microcosm_postgres/encryption/v2/reencryption/utils.py

```diff
@@ -0,0 +1,101 @@
+from dataclasses import dataclass
+from typing import Any, Sequence
+
+from sqlalchemy.inspection import inspect
+from sqlalchemy.orm import Session
+
+from microcosm_postgres.encryption.v2.column import encryption
+from microcosm_postgres.models import Model
+
+
+@dataclass
+class ModelWithEncryption:
```

Comment:
I'm curious why we need a class for this, it seems like we just need it for `encryption_columns` so the class feels a little awkward

Reply:
I see your point.

I guess it was setup to:
* bury some logic.

NB: I've also added another property -> `model_name`.

Given three services are now using this implementation I'm less inclined to update?